packages/notte-integrations/src/notte_integrations/sessions/__init__.py

```
@@ -10,31 +10,41 @@
-from notte_browser.playwright import WindowManager as NotteSessionManager
+from notte_browser.playwright import WindowManager as LocalSessionsManager
 
 from notte_integrations.sessions.anchor import AnchorSessionsManager
 from notte_integrations.sessions.browserbase import BrowserBaseSessionsManager
+from notte_integrations.sessions.hyperbrowser import HyperBrowserSessionsManager
+from notte_integrations.sessions.notte import NotteSessionsManager
 from notte_integrations.sessions.steel import SteelSessionsManager
 
 
-def configure_session_manager(provider: Literal["notte", "steel", "browserbase", "anchor"]) -> None:
+def configure_sessions_manager(
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)